lib/src/header_parser/sub_parsers/var_parser.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:ffigen/src/code_generator.dart';
+import 'package:ffigen/src/header_parser/data.dart';
+import 'package:ffigen/src/header_parser/includer.dart';
+import 'package:logging/logging.dart';
+
+import '../clang_bindings/clang_bindings.dart' as clang_types;
+import '../data.dart';
+import '../utils.dart';
+
+final _logger = Logger('ffigen.header_parser.var_parser');
+
+/// Parses a global variable
+Global? parseVarDeclaration(clang_types.CXCursor cursor) {
+  final type = cursor.type().toCodeGenType();
+  if (type.broadType == BroadType.Unimplemented ||
+      type.getBaseType().isIncompleteStruct ||
+      type.getBaseType().unimplementedReason != null) {
```

Comment:
You can simply check the BroadType, no need to check if unimplementedReason is null. 
```dart
if (type.getBaseType().broadType == BroadType.Unimplemented ||
    type.getBaseType().isIncompleteStruct){
...
}
```